avgratecounter_test.go

```diff
@@ -49,7 +50,7 @@ func TestAvgRateCounterAdvanced(t *testing.T) {
 	time.Sleep(interval - almost)
 	r.Incr(3) // counter = 4, hits = 2
 	check(2.0, 2)
-	time.Sleep(almost)
+	time.Sleep(almost + gap)
```

Comment:
Here is the fix. Without this gap sometimes rate counter counts `r.Incr(1)` from line 48, sometimes - not. Now - always not.